src/response/response.rs

```diff
@@ -132,10 +132,7 @@ impl<B> HttpResponse<B> {
     pub fn del_cookie(&mut self, name: &str) -> usize {
         let headers = self.headers_mut();
 
-        let vals: Vec<HeaderValue> = headers
-            .get_all(header::SET_COOKIE)
-            .map(|v| v.to_owned())
-            .collect();
+        let vals: Vec<HeaderValue> = headers.get_all(header::SET_COOKIE).cloned().collect();
```

Comment:
```suggestion
        let vals: Vec<HeaderValue> = headers
            .get_all(header::SET_COOKIE)
            .map(|v| v.to_owned())
            .collect();
```

Reply:
Is there a reason for that? As far as aI can see, `ToOwned` for both this type and `Cookie` have a blanket implementation which simply calls `Clone`.

While for the `Cookie` example there is no much difference, here we introduce an additional closure.

Reply:
I also see a method [`into_owned`](https://docs.rs/actix-web/3.3.2/actix_web/http/struct.Cookie.html#method.into_owned) for `Cookie`, maybe it's what was supposed to be called instead of `to_owned`?